unfoldingWord/translationCore
# Corrupted Manifest Fix And New Manifest Tests

#### This pull request addresses:
This PR addresses a manifest generation bug, as well as adds tests to help stop future instances of similar bugs happening.


#### How to test this pull request:
Run ```npm test```
Make sure loading functionality is all the same as previously.
Try loading:
online projects,
local (tS/tC) projects,
usfm projects,
use both tools,
new/old projects etc.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/2651)
